### ISSUE TITLE
Chrome crashes when using any of the media keys on my macbook pro

### ISSUE THREAD

**Issue by leroy0211** [beardedspice/beardedspice#980]
Google Chrome crashes whenever I use any of the media keys using Beardedspice. Also when opening the beardedspice icon in the statusbar chrome crashes immediately.  Seems to have started when I've updated my macbook to the latest Catalina version. 

The media keys are still passed along the current running application (in my case spotify). 

Mac version: Catalina 10.15.7 (19H1030)
Chrome version: 90.0.4430.93 
Beardedspice version: 2.2.3

This is the setup I use.
<img width="356" alt="Screenshot 2021-04-28 at 13 40 27" src="https://user-images.githubusercontent.com/2051658/116397851-62389b80-a827-11eb-9130-04d1a8fdc9f4.png">

The applications to track are: 
* Youtube
* Spotify
* Netflix

**Comment by xXLXx** [comment]
I had the same issue with my Google chrome not sure what version of chrome I was before but this happened after the update

![Uploading image.png…]()

**Comment by Stillness-2** [comment]
#899
https://github.com/Stillness-2/beardie/releases

**Comment by xXLXx** [comment]
@Stillness-2 This actually looks good but for some reason I could not make the alpha version work on my M1
![image](https://user-images.githubusercontent.com/4793838/117252000-f7ffa680-ae77-11eb-81e3-2d0cfd0951e0.png)
It just doesnt show up like this like beardedspice does.

I even installed the extension in chrome

**Comment by Stillness-2** [comment]
@xXLXx at least it shows?
![image](https://user-images.githubusercontent.com/12689/117347985-5b0e2f00-aeb2-11eb-8dac-fbcc99cc3692.png)

**Comment by CodeSammich** [comment]
Having the same issue.

**Comment by CodeSammich** [comment]
It seems like Chrome released an update that allows it to access Youtube through media keys. This seems imcompatible with Bearded Spice. Temporary fix is to close the app and use Chrome as is.

**Comment by Stillness-2** [comment]
@CodeSammich https://github.com/Stillness-2/beardie

**Comment by JDTm** [comment]
> @CodeSammich https://github.com/Stillness-2/beardie

a software  which
a) installs a certificate 
b) get access to everything written in a browser 
c) using a "proxy" to see all traffic 

-  is not an option.

**Comment by vityevato** [comment]
> a software which
> a) installs a certificate
> b) get access to everything written in a browser
> c) using a "proxy" to see all traffic
> 
> * is not an option.

Your fears are unfounded.
You can read the source code how and what is used and build from the source yourself if you don't trust.
a) public/private keys for SSL from localhost site, unfortunately, it is necessary because browsers prohibit the extension from communicating using an unencrypted websocket protocol with localhost server
b) BeardedSpice does the same only via AppleScript :)
c) there is no such thing at all

**Comment by xXLXx** [comment]
Update on this @Stillness-2 this actually works on brave browser which is very similar to chrome. BeardedSpice is not working but Beardie does you just need to install the Beardie chrome extension and should be good to go

**Comment by stevyhacker** [comment]
I am having the same crash issue with macOS Big Sur 11.4 and Chrome 91.0.4472.114

**Comment by Delfic** [comment]
My solution was the stop using BeardedSpice
BigSur has this integrated now and I reckon they might have clashed.
Thanks for all the fish :)
![image](https://user-images.githubusercontent.com/4091284/123649089-6716cd00-d821-11eb-9e6d-14346c710f93.png)

It works with media keys, depending on focus.

**Comment by xXLXx** [comment]
@Delfic yeah it does but you cannot use your media controls to say for example you last played on a youtube for music, then media controls should play the youtube and not spotify, right ?

**Comment by Delfic** [comment]
@xXLXx yup, it targets the last media that I manually used or I can choose it on the tray icon before using the shortcuts

**Comment by leroy0211** [comment]
I have disabled beardedspice as well and used the built-in media control of BigSur. The only thing BigSur doesn't have, or at least I couldn't find it, is to pause any playback when unplugging the headphones.
